setup.py

```diff
@@ -103,6 +103,7 @@ def finalize_options(self):
 
 
 model_dir = os.path.join(os.path.dirname(__file__), 'gensim', 'models')
+csparse_dir = os.path.join(os.path.dirname(__file__), 'gensim', 'csparse')
```

Comment:
Can you move this deeply to gensim, for example `gensim.models._csparse` instead of `gensim.csparse` (because this is really low-level internal stuff for special cases).